pkg/client/unversioned/record/event_test.go

```diff
@@ -286,24 +289,24 @@ func TestEventf(t *testing.T) {
 				if !item.expectUpdate {
 					t.Errorf("Expected event create(), got event update()")
 				}
-				called <- struct{}{}
+				wg.Done()
 				return returnEvent, nil
 			},
 		}
 		eventBroadcaster := NewBroadcaster()
 		sinkWatcher := eventBroadcaster.StartRecordingToSink(&testEvents)
 		logWatcher1 := eventBroadcaster.StartLogging(t.Logf) // Prove that it is useful
+		wg.Add(1)
```

Comment:
Instead of adding a wait group, why not just move one of the `<-called` to this line? Isn't that the same?